src/cpp/include/openvino/genai/continuous_batching_pipeline.hpp

```diff
@@ -131,6 +132,7 @@ class OPENVINO_GENAI_EXPORTS ContinuousBatchingPipeline {
 
     GenerationHandle add_request(uint64_t request_id, const ov::Tensor& input_ids, const ov::genai::GenerationConfig& sampling_params);
     GenerationHandle add_request(uint64_t request_id, const std::string& prompt, const ov::genai::GenerationConfig& sampling_params);
+    GenerationHandle add_request(uint64_t request_id, const std::string& prompt, const std::vector<ov::Tensor>& rgbs, const ov::genai::GenerationConfig& sampling_params);
```

Comment:
I'm thinking about `rgbs` argument name. Wouldn't it be more generic to go with something like `images` in case at some point, for some model and so on, we need a different color model?

Reply:
Renamed.